lib/views/after_auth_screens/events/create_event_page.dart

```diff
@@ -164,9 +166,22 @@ class _CreateEventPageState extends State<CreateEventPage> {
                           initialTime: model.eventStartTime,
                         );
 
-                        setState(() {
-                          model.eventStartTime = time;
-                        });
+                        final validationError = Validator.validateEventTime(
+                          time,
+                          model.eventEndTime,
+                        );
+                        final showSnackBar =
+                            navigationService.showTalawaErrorSnackBar;
+                        if (validationError != null) {
+                          showSnackBar(
+                            'Start time must be before end time',
+                            MessageType.error,
```

Comment:
Please write the test for these lines.

Reply:
@noman2002 I tried adding test to expect a talawa error snackbar widget when a wrong start time and end time is choosen but the test is not working.. can you please guide me on how to exactly write test for these lines?

Reply:
It is already implemented somewhere in the code , please have a look.

Reply:
@noman2002 I think it would be better to create a separate issue for this as the main part of the issue has been fixed, all tests have been passed and code coverage is still above 80%